python/modules/DeepTopProducer.py

```diff
@@ -89,13 +138,18 @@ def SelDeepAK8(self, fatj):
     def SelDeepResolved(self, res, jets):
         if math.fabs(res.eta) >= self.etaMax:
             return False
-        if res.discriminator <= self.DeepResolveWP:
-            return False
+        #Moved elsewhere 
+        #if res.discriminator <= self.DeepResolveWP:
+        #    return False
```

Comment:
Do we still need to cut on pt > 1000 for resolve top tagger?

Reply:
its already in, as discussed I put it into the tagger preselection, check the new tagger cfg files to see the cut